writer_test.go

```diff
@@ -1225,6 +1225,27 @@ func ExampleMediaPlaylist_Segments_scte35_oatcls() {
 	// media2.ts
 }
 
+func ExampleMediaPlaylist_Segments_scte35_cue() {
+	f, _ := os.Open("sample-playlists/media-playlist-with-cue-scte35.m3u8")
+	p, _, _ := DecodeFrom(bufio.NewReader(f), true)
```

Comment:
[ioutil.ReadFile](https://pkg.go.dev/io/ioutil#ReadFile) would be a better approach than os.Open+bufio.NewReader... Also we should not ignore these possible (even if unlikely) errors.

Reply:
Replace it with ioutil.Read() and added error handling in: https://github.com/zencoder/m3u8/pull/8/commits/c953678b1aa9fccf3c8ab6bb7b540872f75ba4c0